SECURITY.md

```diff
@@ -0,0 +1,3 @@
+#Security Policy
```

Comment:
I think a space is needed here, it is not visualising correctly in Github viewer.

Reply:
fixed it.

Reply:
fixed it.